cmd/root.go

```diff
@@ -616,47 +345,49 @@ func updateIssue(ghIssue github.Issue, jIssue jira.Issue, ghClient github.Client
 
 // createIssue generates a JIRA issue from the various fields on the given GitHub issue, then
 // sends it to the JIRA API.
-func createIssue(issue github.Issue, ghClient github.Client, jClient jira.Client) error {
+func createIssue(config lib.Config, issue github.Issue, ghClient github.Client, jClient jira.Client) error {
```

Comment:
same: all of these functions that are *not* directly concerned with configuring and launching the command should be outside of this file and package. this function should be in lib.